@navikt/ds-css/loader.css

```diff
@@ -1,3 +1,15 @@
+:root {
+  --navds-loader-color-foreground: var(--navds-color-gray-40);
+  --navds-loader-color-background: var(--navds-color-gray-10);
+  --navds-loader-color-neutral-foreground: var(--navds-color-gray-40);
+  --navds-loader-color-interaction-foreground: var(--navds-color-blue-50);
+  --navds-loader-color-inverted-foreground: var(--navds-color-white);
+  --navds-loader-color-inverted-background: rgba(255, 255, 255, 0.3);
+  --navds-loader-color-transparent-background: transparent;
+  --navds-loader-button-color-background: rgba(255, 255, 255, 0.3);
```

Comment:
Hvordan håndtere vi disse tilfellene?

Reply:
Om det kun er for `loader` enn så lenge, så tenker jeg at inline er greit. Om vi ser behov for gjennomsiktige gråfarger flere steder, så kan vi vurdere å trekke det opp til den globale paletten.